### PR TITLE
fix(packages/sui-mono): fix release for monopackage

### PR DESCRIPTION
**Pull request by rmoralp**
## Description
Fix release error when repo is monopackage (with no workspaces)
`✖ Error: Error: Root/package.json: ENOENT: no such file or directory, open 'Root/package.json'`

### CODE REVIEW

File: packages/sui-mono/src/config.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const CWD = process.cwd()
 const packageFile = getPackageJson(CWD)
 
 const getWorkspaces = workspaces => {
-  if (workspaces.length === 0) return ['Root']
```
Comment (jcger) [inline packages/sui-mono/src/config.js:9]: isn't this gonna break the commit scopes if there as be no change in a workspace?

Comment (rmoralp) [reply packages/sui-mono/src/config.js:9]: I think that `getWorkspaces` function should always the same, only return a list (array) of valid paths.
I have checked and fixed all side effects.

Comment (rmoralp) [reply packages/sui-mono/src/config.js:9]: You could check all `getWorkspaces` usages here:
 https://github.com/SUI-Components/sui/search?q=getWorkspaces&type=code

**Review by jcger — APPROVED** [review]

**Comment by rmoralp** [comment]
Beta published: `@s-ui/mono@2.22.0-beta.0`

**Review by kikoruiz — APPROVED** [review]

**Review by jordevo — APPROVED** [review]

**Pull request merged**